runtime/wasm/src/host_exports.rs

```diff
@@ -377,7 +377,7 @@ where
         callback: &str,
         user_data: store::Value,
         flags: Vec<String>,
-    ) -> Result<Vec<EntityOperation>, HostExportError<impl ExportError>> {
```

Comment:
I think the comment above this method could use updating now because it only mentions entity operations.